test/built-ins/Symbol/species/Symbol.species.in_Array.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ Array has a property at `Symbol.species`
+es6id: 22.1.2.5
```

Comment:
In order to be consistent with tests for `Symbol.iterator`, I believe these `Symbol.species.in_*.js` should go on their respective folders, like `test/built-ins/Array/prototype/Symbol.species.js` on this case.